docs/source/sysadmin_guide/configure_sgx.rst

```diff
@@ -0,0 +1,458 @@
+
+.. code-block:: console
+
+    $ cd ~/sgx
+    $ wget https://download.01.org/intel-sgx/linux-1.8/sgx_linux_ubuntu16.04.1_x64_psw_1.8.100.37689.bin
+    $ chmod +x sgx_linux_ubuntu16.04.1_x64_psw_1.8.100.37689.bin
+    $ sudo ./sgx_linux_ubuntu16.04.1_x64_psw_1.8.100.37689.bin
+
+Check to make sure the kernel module is loaded:
+
+.. code-block:: console
+
+    $ lsmod | grep sgx
+    isgx                   36864  2
+
+If the output does not show the isgx module, make sure that
+SGX is set to "Enabled" in the BIOS.
+
+If you're still having trouble, the SGX software may need to be reinstalled:
+
+.. code-block:: console
+
+    $ sudo /opt/intel/sgxpsw/uninstall.sh
+    $ cd ~/sgx
+    $ sudo ./sgx_linux_x64_driver_4b57088.bin
+    $ sudo ./sgx_linux_ubuntu16.04.1_x64_psw_1.8.100.37689.bin
+
+After ensuring that the SGX kernel module is loaded, go to the next section
+to install and configure Sawtooth.
+
+
+Configuring Sawtooth to Use SGX
+===============================
+
+Install Sawtooth
+----------------
+
+.. code-block:: console
+
+    $ sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 8AA7AF1F1091A5FD
+    $ sudo add-apt-repository 'deb http://repo.sawtooth.me/ubuntu/0.8/stable xenial universe'
+    $ sudo apt-get update
+    $ sudo apt-get install -y -q \
+      sawtooth \
+      python3-sawtooth-poet-sgx
+
+
+Configure the validator to use SGX PoET
+---------------------------------------
+
+After installing Sawtooth, add config settings so PoET will work properly.
+Create the file ``/etc/sawtooth/poet_enclave_sgx.toml``
+with your favorite editor (such as vi):
+
+.. code-block:: console
+
+    $ sudo vi /etc/sawtooth/poet_enclave_sgx.toml
+
+Add the following lines:
+
+.. code-block:: console
+
+    # spid is a 32-digit hex string tied to the enclave implementation
+
+    spid = 'B785C58B77152CBE7FD55EE3851C4990'
+
+    # ias_url is the URL of the Intel Attestation Service (IAS) server.
+
+    ias_url = 'https://test-as.sgx.trustedservices.intel.com:443'
+
+    # spid_cert_file is the full path to the PEM-encoded certificate file that was
+    # submitted to Intel in order to obtain a SPID
+
+    spid_cert_file = '/etc/sawtooth/maiden-lane-poet-linkable-quotes.pem'
+
+Next, copy the .pem certificate file you downloaded earlier.
+Replace [example] in the path below with the path to the certificate file on
+your local system:
+
+.. code-block:: console
+
+    $ sudo cp /[example]/maiden-lane-poet-linkable-quotes.pem /etc/sawtooth/maiden-lane-poet-linkable-quotes.pem
+
+Become the sawtooth user and change to ``/tmp``:
+
+.. code-block:: console
+
+    $ sudo -u sawtooth -s
+    $ cd /tmp
+
+Create validator keys:
+
+.. code-block:: console
+
+    $ sawtooth admin keygen
+
+.. note:: If you're configuring multiple validators, the steps below 
+    do not need to be repeated on subsequent validators. There is some
+    additional configuration required though. See 
```

Comment:
Does this mean to skip to the "Validator config file" section?  Shouldn't you log out of the sawtooth account first? If so, I'd recommend this wording:

     If you're configuring multiple validators, the steps below are required 
     for the first validator only.  For additional validators, you can 
     skip to the `$ exit` command to log out of the sawtooth account, 
     then continue with :ref:`val-config`.